src/test/java/testsuite/failover/NetworkFailuresFailoverIntegrationTest.java

```diff
@@ -90,8 +91,8 @@
   private static final String DB_CONN_CLUSTER_RO = "jdbc:mysql:aws://" + DB_HOST_CLUSTER_RO + "/" + DB_DATABASE;
   private static final String DB_CONN_INSTANCE_PATTERN = "jdbc:mysql:aws://" + DB_HOST_INSTANCE_PATTERN + "/" + DB_DATABASE;
 
-  private static final String DB_USER = System.getProperty("com.mysql.cj.testsuite.failover.networkFailures.user");
```

Comment:
Do we know why it was called com.mysql.cj.testsuite.failover.networkFailures.user?

Reply:
There's another set of user credentials for regression tests. Failover set of tests uses their own credentials.

Reply:
Just old parameters. There was also a ticket (389) to have them updated to use the same ones as the other Failover Tests